src/com/leetcode/problems/CalculateMoney.java

```diff
@@ -0,0 +1,25 @@
+package com.leetcode.problems;
+
+import java.util.Arrays;
+
+public class CalculateMoney {
+  public static void main(String[] args) {
+    System.out.println(totalMoney(20));
+  }
+
+  public static int totalMoney(int n) {
+    int sum = 0, moneyAdded = 1, dayCounter = 0, week = 1;
+    for (int i = 0; i < n; i++) {
+      sum += moneyAdded++;
+      dayCounter++;
+      if (dayCounter % 7 == 0) {
+        week++;
+        moneyAdded = week;
+        dayCounter = 0;
+      }
+    }
+    return sum;
+  }
+}
+
+//https://leetcode.com/problems/calculate-money-in-leetcode-bank/
```

Comment:
Every file must have an empty new line at the end of it. If you'd configure a proper formatter in your IDE it would take care of this automatically. Move this URL to the top, just before class declaration. 
Apply this to all pull requests.  

Please make sure you always add the problem URL as a comment.

Reply:
Also.. place a space between "//" and actual comment..